examples/Extensions/AddSitelinks.php

```diff
@@ -0,0 +1,339 @@
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID
+     * @param int $campaignId the campaign ID
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $campaignId
+    ) {
+        $campaignResourceName = ResourceNames::forCampaign($customerId, $campaignId);
+
+        // Creates extension feed items as sitelinks.
+        $extensionFeedItems =
+            self::createExtensionFeedItems($googleAdsClient, $customerId, $campaignResourceName);
+
+        // Creates a campaign extension setting using the previously created extension feed items.
+        $campaignExtensionSetting = new CampaignExtensionSetting([
+            'campaign' => new StringValue(['value' => $campaignResourceName]),
+            'extension_type' => ExtensionType::SITELINK,
+            'extension_feed_items' => $extensionFeedItems
+        ]);
+
+        // Creates the campaign extension setting operation.
+        $campaignExtensionSettingOperation = new CampaignExtensionSettingOperation();
+        $campaignExtensionSettingOperation->setCreate($campaignExtensionSetting);
+
+        // Issues a mutate request to add the campaign extension setting.
+        $campaignExtensionSettingServiceClient =
+            $googleAdsClient->getCampaignExtensionSettingServiceClient();
+        $response = $campaignExtensionSettingServiceClient->mutateCampaignExtensionSettings(
+            $customerId,
+            [$campaignExtensionSettingOperation]
+        );
+
+        // Prints the resource name of the created campaign extension setting.
+        /** @var CampaignExtensionSetting $addedCampaignExtensionSetting */
+        $addedCampaignExtensionSetting = $response->getResults()[0];
+        printf(
+            "Created a campaign extension setting with resource name: '%s'%s",
+            $addedCampaignExtensionSetting->getResourceName(),
+            PHP_EOL
+        );
+    }
+
+    /**
+     * Creates a list of extension feed items.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID
+     * @param string $campaignResourceName the resource name of the campaign to target
+     * @return StringValue[] the list of extension feed items' resource names
+     */
+    private static function createExtensionFeedItems(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $campaignResourceName
+    ) {
+        // Creates a site link feed item 1.
```

Comment:
nit: "**the first** site link feed item" instead of "a site link feed item **1**"?

Same with the other instances